# Rob_Params:
#   num_rob: 2
#
# smaller
Blob_Params:
  threshold_step: 6 #10
  min_threshold: 200 #224 #25
  max_threshold: 255
  min_repeatability: 4 #4
  min_dist_between_blobs: 5 #5
  filter_by_color: true
  blob_color: 255
  filter_by_area: true
  min_area: 4 #20
  max_area: 2083
  filter_by_circularity: false
  min_circularity: 0.3
  max_circularity: 1
  filter_by_inertia: false
  min_inertia_ratio: 0.2
  max_inertia_ratio: 1
  filter_by_convexity: false
  min_convexity: 0.95
  max_convexity: 1
